app/views/reservations/_account_field.html.haml

```diff
@@ -1,7 +1,7 @@
 .well
   .container
     .row
-      .span6
+      .col-md-6
```

Comment:
in line 2 there's a `container` div we should remove because it's making children to overflow the well container (hierarchy is container > row >  column)